### PR TITLE
Review

### PR DESCRIPTION
**Pull request by Yakushev-Aleksandr**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,99 @@
...
 
 const App: React.FC = () => {
+  const [showEdit, setShowEdit] = useState<boolean>(false);
+  const [edit, setEdit] = useState<string>("");
+  const [editId, setEditId] = useState<number>();
+  const [saveEdit, setSaveEdit] = useState<string>("");
+
+  const dispatch = useAppDispatch();
+
+  const addTodosHandler = (initialState: IItem) => {
```
Comment (evl-alex) [inline src/App.tsx]: it is not initial state better to name it like "todoItem"

File: src/App.tsx [hunk 2]
```diff
@@ -1,17 +1,99 @@
...
+    setEditId(initialState.id);
+    dispatch(addItem(initialState));
+  };
+
+  const todo = useAppSelector(selectItem);
+
+  const editElementHandler = (initialState: IItem) => {
```
Comment (evl-alex) [inline src/App.tsx]: same here

File: src/App.tsx [hunk 3]
```diff
@@ -1,17 +1,99 @@
...
+    setShowEdit((previousState: boolean) => !previousState);
+    setEditId(initialState.id);
+
+    setEdit(initialState.todo);
+    setSaveEdit(initialState.todo);
+  };
+
+  const deleteElementHandler = (initialState: IItem) => {
```
Comment (evl-alex) [inline src/App.tsx]: same here

File: src/App.tsx [hunk 4]
```diff
@@ -1,17 +1,99 @@
...
+  };
+
+  // МОДАЛЬНОЕ ОКНО
+  //__________________________________________________________
+  const editValueHandler = (textInput: string) => {
+    setEdit(textInput);
+  };
+
+  const cancelShowHandler = () => {
+    setShowEdit((previousState: boolean) => false);
```
Comment (evl-alex) [inline src/App.tsx]: as you don't use previousState, there is no need to declare it, can be simplified to usetShowEdit(() => false) or just usetShowEdit(false)

File: src/Features/textItem/item-slice.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    editItem: (state, actions: PayloadAction<{ id: number; todo: string }>) => {
+      state.find((item): number | string | undefined =>
+        item.id === actions.payload.id ? (item.todo = actions.payload.todo) : ""
+      );
+    },
+    deleteItem: (state, actions: PayloadAction<IItem>) => {
+      return state.filter((todo) => actions.payload.id !== todo.id);
+    },
+    completedItem: (
```
Comment (evl-alex) [inline src/Features/textItem/item-slice.ts]: completeItem

**Review by Yakushev-Aleksandr — COMMENTED** [review]
I added Muo library

**Pull request merged**